atomicapp/nulecule/main.py

```diff
@@ -90,6 +90,16 @@ def __init__(self, app_spec, destination=None, answers_file=None):
             if os.path.isfile(os.path.join(self.app_path, ANSWERS_FILE)):
                 self.answers_file = os.path.join(self.app_path, ANSWERS_FILE)
 
+        # TODO: put this in a better place in the future.
```

Comment:
Better place could actually just be the `openshift.py` under providers.

I've been thinking lately of refactoring how we use each provider since we're adding new commands in the future. For example, changing `def undeploy` to `def stop` and `def deploy` to `def run`. Making it a bit more modular for new providers to be added. 

Perhaps a `def check_host` or something similar could help